pyproject.toml

```diff
@@ -27,3 +27,6 @@ disable = ["raw-checker-failed",
         "too-many-statements",
         "too-many-instance-attributes"]
 
+[tool.mypy]
+ignore_missing_imports = true
+check_untyped_defs = true
```

Comment:
Shouldn't untyped defs be disallowed (disallow_untyped_defs)?

Reply:
We agreed not to enforce yet.